stages/org.osbuild.systemd

```diff
@@ -73,6 +87,21 @@ def main(tree, options):
     if default_target:
         subprocess.run(["systemctl", "--root", tree, "set-default", default_target], check=True)
 
+    if presets:
+        svcs = {}
+
+        for svc in svcs:
+            # verify that we don't have duplicate service names with different
+            # states
+            if svc["name"] in svcs:
+                raise RuntimeError(f"{svc['name']!r} has multiple defined states.")
+
+            svcs[svc["name"]] = svc["state"]
+
+        # XXX, or do we want /etc? this might problematic for ostree
+        with open(f"{tree}/usr/lib/systemd/system-preset/50-osbuild.conf", "w", encoding="utf8") as f:
+            f.writelines(f"{name} {state}\n" for name, state in svcs.items())
```

Comment:
this should be "state service" e.g. "enable sshd.service" https://www.freedesktop.org/software/systemd/man/systemd.preset.html#Preset%20File%20Format